next-sitemap.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('next-sitemap').IConfig} */
+module.exports = {
+  siteUrl: 'https://authzed.com/docs',
```

Comment:
should we be getting this from an environment variable?

Reply:
Updated to read the existing BASE_DIR env var but we don't use an env var for the host.